ee/server/configuration/oauth.ts

```diff
@@ -0,0 +1,92 @@
+import { capitalize } from '@rocket.chat/string-helpers';
+
+import { OAuthEEManager } from '../lib/oauth/Manager';
+import { onLicense } from '../../app/license/server';
+import { callbacks } from '../../../app/callbacks/server';
+import { settings } from '../../../app/settings/server';
+import { Logger } from '../../../app/logger/server';
+
+interface IUserService {
+	serviceName: string;
+	serviceData: Record<string, any>;
+	user: Record<string, any>;
+}
+
+interface IUserIdentity {
+	serviceName: string;
+	identity: Record<string, any>;
+	user: Record<string, any>;
+}
+
+interface ISettings {
+	mapChannels: string;
+	mergeRoles: string;
+	rolesClaim: string;
+	groupsClaim: string;
+	channelsAdmin: string;
+	channelsMap: string;
+}
```

Comment:
I would rename the interfaces to `IOAuthUserService`, `IOAuthUserIdentity` and `IOAuthSettings` to avoid confusion between them and other existing interfaces with similar names like `ISetting`